dapps/profile.vue/src/i18n/en.json

```
@@ -492,6 +492,8 @@
       "transactions": {
         "charged-at": "charged at",
         "credit-charged": "Credit recharged",
+        "credit-running-recharge": "Credit recharge not yet completed",
+        "credit-failed-recharge": "The charging has unfortunately failed",
```

```suggestion
        "credit-failed-charge": "The charging has failed unfortunately",
```
Question #2 from above applies here too.